src/components/Errors/Error.tsx

```diff
@@ -0,0 +1,29 @@
+/* eslint-disable jsx-a11y/control-has-associated-label */
+import React from 'react';
+import classNames from 'classnames';
+
+type Props = {
+  errorMessage: string,
+  removeMessage: () => void,
+};
+
+export const Error: React.FC<Props> = (
+  { errorMessage, removeMessage },
+) => {
+  const isError = errorMessage !== '';
+
+  return (
+    <div className={classNames(
+      'notification is-danger is-light has-text-weight-normal',
+      { hidden: !isError },
+    )}
+    >
+      <button
+        type="button"
+        className="delete"
+        onClick={() => removeMessage()}
```

Comment:
You can just pass the function name if you should call it without any arguments:
```
onClick={removeMessage}
```
It looks better